unattended_scripts/unattended_installation.sh

```diff
@@ -1,19 +1,27 @@
 #!/bin/bash
 
+# Wazuh installer
+# Copyright (C) 2015-2021, Wazuh Inc.
+#
+# This program is a free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public
+# License (version 2) as published by the FSF - Free Software
+# Foundation.
+
 ## Package vars
-WAZUH_MAJOR="4.2"
-WAZUH_VER="4.2.5"
-WAZUH_REV="1"
-ELK_VER="7.10.2"
-ELKB_VER="7.12.1"
-OD_VER="1.13.2"
-OD_REV="1"
-WAZUH_KIB_PLUG_REV="1"
+wazuh_major="4.2"
+wazuh_ver="4.2.5"
+wazuh_rev="1"
+elk_ver="7.10.2"
+elkb_ver="7.12.1"
+od_ver="1.13.2"
+od_rev="1"
+wazuh_kib_plug_rev="1"
```

Comment:
```suggestion
wazuh_major="4.2"
wazuh_version="4.2.5"
wazuh_revision="1"
elastic_oss_version="7.10.2"
elastic_basic_version="7.12.1"
opendistro_version="1.13.2"
opendistro_revision="1"
wazuh_kibana_plugin_revision="1"
```

Please replace all variable names in all involved files.

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/1043/commits/e77ab7963be712dea06bebb2e1204a24850db201